rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/QrdaControllerV1.java

```diff
@@ -56,15 +58,26 @@ public QrdaControllerV1(final QrdaService qrdaService, final ValidationService v
 	 * Endpoint to transform an uploaded file into a valid or error json response
 	 *
 	 * @param file Uploaded file
+	 * @param test Whether the conversion is a test or not
 	 * @return Valid json or error json content
 	 */
 	@PostMapping(headers = {"Accept=" + Constants.V1_API_ACCEPT})
-	public ResponseEntity<String> uploadQrdaFile(@RequestParam MultipartFile file) {
+	public ResponseEntity<String> uploadQrdaFile(@RequestParam MultipartFile file,
+			@RequestHeader(required = false, name = "Purpose") String purpose) {
 		String originalFilename = file.getOriginalFilename();
-		API_LOG.info("Conversion request received");
+
+		if (!StringUtils.isEmpty(purpose)) {
+			if (purpose.length() > 25) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 25 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)